src/forsyde/ut_process_constructors.hpp

```diff
@@ -1741,6 +1741,85 @@ class fanout : public ut_process
 #endif
 };
 
+<<<<<<< HEAD
+=======
+template <class T1, class T2, class TCS>
+class zipU : public ut_process
+{
+public:
+    UT_in<T1> iport1;        ///< port for the input channel 1
+    UT_in<T2> iport2;        ///< port for the input channel 2
+    UT_in<TCS> controlport;   ///< port for the control signal   
+    UT_out<std::tuple<std::vector<T1>,std::vector<T2>>> oport1;///< port for the output channel
+
+    typedef std::function<unsigned int(const TCS&)> gamma_functype;
```

Comment:
`size_t`

Reply:
The common convention in ForSyDe-SystemC is to have the return value as the first argument called by reference.